Source/UnitTests/Common/BitUtilsTest.cpp

```diff
@@ -71,21 +71,21 @@ TEST(BitUtils, IsValidLowMask)
 
   EXPECT_TRUE(Common::IsValidLowMask((u8)~0b0));
   EXPECT_FALSE(Common::IsValidLowMask((u8)(~0b0 - 1)));
-  EXPECT_FALSE(Common::IsValidLowMask((u8)~(0b10000)));
+  EXPECT_FALSE(Common::IsValidLowMask((u8) ~(0b10000)));
```

Comment:
Sure can. Here's a patch for the file that applies to the branch. (Had to rename the extension, as it doesn't accept .patch files.)

[0001-Use-static_cast-in-BitUtilsTest.txt](https://github.com/dolphin-emu/dolphin/files/1115758/0001-Use-static_cast-in-BitUtilsTest.txt)